### DISCUSSION TITLE
Sidebar Navigation for Files, Trash and Search

### DISCUSSION THREAD

**Discussion by haxzie** [haxzie/snipp.in#25] [Ideas]
Hey folks! Would love some feedback on having a side bar like VSCODE to introduce search and settings options.
@FlorianWoelki @prikeshsavla :raised_hands: 

![Frame 1(31)](https://user-images.githubusercontent.com/53584487/103744642-22556a80-5024-11eb-8249-2ab9fd75de86.png)

**Comment by FlorianWoelki** [comment]
I love it :) Looks pretty awesome!

**Comment by haxzie** [comment]
Released via #34  :confetti_ball:
